astropy/io/fits/hdu/compressed/header.py

```diff
@@ -746,9 +630,15 @@ def _image_header_to_empty_bintable(
 
         if card.keyword == "":
             bintable.header.add_blank()
+        elif _is_reserved_keyword(card.keyword):
+            warnings.warn(
+                f"Keyword {card.keyword!r} is reserved "
+                "for use by the FITS Tiled Image "
+                "Convention so will be ignored"
+            )
```

Comment:
This could be a `VerifyWarning` as done in other HDU classes ?

Reply:
Done